scripts/irods/test/test_rulebase.py

```diff
@@ -435,26 +482,50 @@ def make_pep_rule(self, pep_name, rule_body):
 
         return '{pep_name} {{ {write_statements};{rule_body} }}'.format(**locals())
 
-    def make_new_server_config_json(self, server_config_filename):
-        # load server_config.json to inject a new rule base
-        with open(server_config_filename) as f:
-            svr_cfg = json.load(f)
-
-        # inject a new rule base into the native rule engine
-        svr_cfg['plugin_configuration']['rule_engines'][0]['plugin_specific_configuration']['re_rulebase_set'] = ["test", "core"]
-
-        # dump to a string to repave the existing server_config.json
-        return json.dumps(svr_cfg, sort_keys=True,indent=4, separators=(',', ': '))
+       
+    def make_pep_rule_python(self, pep_name, rule_body):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/trel/irods/pullRequest?prid=4593032)